<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newProfessor</class>
 <widget class="QDialog" name="newProfessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1395</width>
    <height>547</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="leftMenu" native="true">
        <property name="styleSheet">
         <string notr="true">#leftMenu
{
	
	background-color: rgb(254, 191, 78);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="leftMenuSub" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QLabel" name="logo">
                 <property name="maximumSize">
                  <size>
                   <width>166</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#logo
{
	backgroud-repeat: no-repeat;
backgroud-position: center;
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>../imgplural/nova.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QFrame" name="frame_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QPushButton" name="homeButton">
                 <property name="toolTip">
                  <string>Home</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#homeButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	padding: 2px 10px;
	margin: 0;
	text-align: left;
	margin-top: 30px
}</string>
                 </property>
                 <property name="text">
                  <string>Home</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/home.svg</normaloff>../imgplural/feather/home.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="finButton">
                 <property name="toolTip">
                  <string>Financeiro</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#finButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	padding: 2px 10px;
	margin: 0;
	text-align: left;
	margin-top: 30px
}</string>
                 </property>
                 <property name="text">
                  <string>Financeiro</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/dollar-sign.svg</normaloff>../imgplural/feather/dollar-sign.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="admButton">
                 <property name="toolTip">
                  <string>Administrativo</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#admButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	margin: 0;
	text-align: left;
	padding: 2px 10px;
	margin-top: 30px
}</string>
                 </property>
                 <property name="text">
                  <string>Administrativo</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/archive.svg</normaloff>../imgplural/feather/archive.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QPushButton" name="infoButton">
                 <property name="toolTip">
                  <string>Help</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#infoButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	padding: 2px 10px;
	margin: 0;
	text-align: left;
}</string>
                 </property>
                 <property name="text">
                  <string>Informações</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/info.svg</normaloff>../imgplural/feather/info.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="helpButton">
                 <property name="toolTip">
                  <string>Help</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#helpButton
{
	border: none;
	backgroud-color: transparent;
	backgroud: none;
	padding: 2px 10px;
	margin: 0;
	text-align: left;
	margin-top: 20px
}</string>
                 </property>
                 <property name="text">
                  <string>Help</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../imgplural/feather/help-circle.svg</normaloff>../imgplural/feather/help-circle.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightMenu" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#rightMenu
{
	
	background-color: rgb(26, 35, 25);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>30</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#label_6
{
	text-align: center;
	color: rgb(255, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Novo PROF</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QFrame" name="frame_11">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="labelNome">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">#labelNome
{
	color: rgb(255, 255, 255)
}</string>
                    </property>
                    <property name="text">
                     <string>Nome</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditNome">
                    <property name="styleSheet">
                     <string notr="true">#lineEditNome
{
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	
	color: rgb(100, 100, 100);
}

#lineEditNome:focus
{
	
	color: rgb(200, 200, 200);
}
</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelIdade">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">#labelIdade

{
	color: rgb(255, 255, 255)
}</string>
                    </property>
                    <property name="text">
                     <string>Idade</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditIdade">
                    <property name="styleSheet">
                     <string notr="true">#lineEditIdade
{
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	
	color: rgb(100, 100, 100);
}

#lineEditIdade:focus
{
	
	color: rgb(200, 200, 200);
}
</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_7">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="labelTelefone">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">#labelTelefone

{
	color: rgb(255, 255, 255)
}</string>
                    </property>
                    <property name="text">
                     <string>Telefone</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditTelefone">
                    <property name="styleSheet">
                     <string notr="true">#lineEditTelefone
{
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	
	color: rgb(100, 100, 100);
}

#lineEditTelefone:focus
{
	
	color: rgb(200, 200, 200);
}
</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelEmail">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">#labelEmail

{
	color: rgb(255, 255, 255)
}</string>
                    </property>
                    <property name="text">
                     <string>Email</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditEmail">
                    <property name="styleSheet">
                     <string notr="true">#lineEditEmail
{
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	
	color: rgb(100, 100, 100);
}

#lineEditEmail:focus
{
	
	color: rgb(200, 200, 200);
}
</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_5">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="labelTurma">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">#labelTurma
{
	color: rgb(255, 255, 255)
}</string>
                    </property>
                    <property name="text">
                     <string>Materia</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditTurma">
                    <property name="styleSheet">
                     <string notr="true">#lineEditTurma
{
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	
	color: rgb(100, 100, 100);
}

#lineEditTurma:focus
{
	
	color: rgb(200, 200, 200);
}
</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelMensal">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">#labelMensal
{
	color: rgb(255, 255, 255)
}</string>
                    </property>
                    <property name="text">
                     <string>Salário</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditMensal">
                    <property name="styleSheet">
                     <string notr="true">#lineEditMensal
{
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);
	
	color: rgb(100, 100, 100);
}

#lineEditMensal:focus
{
	
	color: rgb(200, 200, 200);
}
</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_12">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QPushButton" name="addAlunoBtn">
                    <property name="styleSheet">
                     <string notr="true">#addAlunoBtn
{
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
	
	color: rgb(200, 200, 200);
}

#addAlunoBtn:hover
{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}

#addAlunoBtn:pressed
{
	background-color: rgb(200, 200, 200);

	
	color: rgb(35, 35, 35);
}</string>
                    </property>
                    <property name="text">
                     <string>Confirmar</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="cancelarAlunoBtn">
                    <property name="styleSheet">
                     <string notr="true">#cancelarAlunoBtn
{
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
	
	color: rgb(200, 200, 200);
}

##cancelarAlunoBtn:hover
{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}

##cancelarAlunoBtn:pressed
{
	background-color: rgb(200, 200, 200);

	
	color: rgb(35, 35, 35);
}</string>
                    </property>
                    <property name="text">
                     <string>Cancelar</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
